### ISSUE TITLE
explicitly state you must clone dotfiles to ~

### ISSUE THREAD

**Issue by abaird** [thoughtbot/dotfiles#488]
I tried to follow the directions [here](https://github.com/thoughtbot/dotfiles#install) and could not get it installed because I had my clone of dotfiles not located in the home directory. If there is no other way to get this to work correctly, you should explicitly state this, changing this line:

"Clone onto your laptop:"
to
"Clone onto your laptop in your home directory:"

This is very confusing to new users that misunderstand this because when you run `rcup` there is no helpful message telling you what's going on, it just returns nothing.

**Comment by geoffharcourt** [comment]
The later step ("Install the dotfiles") references a dotfiles directory under `$HOME`. If you're choosing a different location for the repository that location for the ENV declaration needs to change, but I'm having trouble finding a succinct way to express that other than just providing the example.

**Comment by brandoncordell** [comment]
Can we just change the example to `git clone git://github.com/thoughtbot/dotfiles.git ~/dotfiles`? Explicit and concise.

**Comment by abaird** [comment]
@geoffharcourt - maybe it's because I'm a little dense, but when I first came to the project, I wasn't sure if rcm was creating the dotfiles directory for me or if I needed to create it myself. I went looking in the docs for that and got even more confused by [this](http://thoughtbot.github.io/rcm/rcm.7.html) which wants the dotfile directory to be .dotfiles (and not dotfiles).

I understand this better now, but when you are new to dotfile management it's hard to tell what way is up. I think it's easier for noobz like me to follow a well defined script to get the idea of what's going on and then learn how to extend it. 

I do like @brandoncordell 's suggestion.
